up-l1/README.adoc

```diff
@@ -154,20 +130,18 @@ registerListener(
 | The listener to be registered.
 
 | result
-| Future<Void, link:../basics/error_model.adoc[UStatus]>
+| Future<Void>
 | A succeeded future if the listener has been registered successfully. +
-Otherwise, a failed future containing a `UStatus` indicating the reason for the failure.
+Otherwise, a failed future which contains a `UCode` indicating the reason for the failure.
 |===
 
-[oft-sid="req~utransport-register-listener~1"]
---
 Implementations
 
+* *MAY* fail invocations with a `UCode::UNIMPLEMENTED` if the transport does not support the _push_ <<_delivery_method>>. In that case, the <<_unregisterlistener>> method *MUST* also fail accordingly.
```

Comment:
Something about the `_` gets mangled or something...?

![image](https://github.com/eclipse-uprotocol/up-spec/assets/11622119/2ee1801b-4732-472c-a427-7bed21ed0be6)

Reply:
That's weird, the vscode preview correctly renders the links and it is the standard AsciiDoc format. Anyway, I will replace it with the (linked to) section title.